setup.py

```diff
@@ -1,179 +0,0 @@
-                log.warn(f"Removing {self.TARGET_DIR} and re downloading the UI.")
-                shutil.rmtree(self.TARGET_DIR)
-            else:
-                log.warn(f"Static directory {self.TARGET_DIR} already exists, skipping. ")
-                return
-
-        with tempfile.NamedTemporaryFile() as download_file:
-            log.info(f"Downloading UI distribution to temporary file: {download_file.name}")
-
-            if self.ALTERNATE_UI_DOWNLOAD_URL:
-                log.info(f"Downloading UI from {self.ALTERNATE_UI_DOWNLOAD_URL}")
-                self._download_tarball(self.ALTERNATE_UI_DOWNLOAD_URL, download_file)
-            else:
-                log.info(f"Attempting to download UI for version {version}")
-                try:
-                    self._download_tarball(self.UI_DOWNLOAD_URL, download_file)
-                except urllib.error.HTTPError:
-                    log.warn(f"Failed to retrieve UI for {version}. Downloading latest UI.")
-                    self._download_tarball(self.DEV_UI_DOWNLOAD_URL, download_file)
-
-            log.info(f"Extracting UI static files to {self.TARGET_DIR}")
-            with tarfile.open(fileobj=download_file) as tfp:
-                tfp.extractall(self.TARGET_DIR)
-
-    def _download_tarball(self, url, download_file):
-        urllib.request.urlretrieve(url, filename=download_file.name)
-
-
-class SDistCommand(_SDistCommand):
-    def run(self):
-        self.run_command("prepare_static")
-        return super().run()
-
-
-class BuildPyCommand(_BuildPyCommand):
-    def run(self):
-        self.run_command("prepare_static")
-        return super().run()
-
-
-django_ansible_base_branch = os.getenv('DJANGO_ANSIBLE_BASE_BRANCH', '2025.3.7')
-django_ansible_base_dependency = (
-    'django-ansible-base[jwt-consumer,feature-flags] @ '
-    f'git+https://github.com/ansible/django-ansible-base@{django_ansible_base_branch}'
-)
-
-requirements = [
-    "galaxy-importer>=0.4.27,<0.5.0",
-    "pulpcore>=3.49.0,<3.50.0",
-    "pulp_ansible==0.23.1",
-    "pulp-container>=2.19.2,<2.20.0",
-    "django-prometheus>=2.0.0",
-    "social-auth-core>=4.4.2",
-    "social-auth-app-django>=5.2.0",
-    "django-auth-ldap==4.0.0",
-    "drf-spectacular",
-    "dynaconf>=3.2.10",
-    "insights_analytics_collector>=0.3.0",
-    "boto3",
-    "distro",
-    "django-flags>=5.0.13",
-    django_ansible_base_dependency,  # noqa 501
-    "django-crum==0.7.9",
-    # From vendored automated_logging
-    "marshmallow<4.0.0,>=3.6.1",
-    "django-picklefield<4.0.0,>=3.0.1",
-    "django-ipware<4.0.0,>=3.0.0",
-]
-
-
-# https://softwareengineering.stackexchange.com/questions/223634/what-is-meant-by-now-you-have-two-problems
-def strip_package_name(spec):
-    operators = ["=", ">", "<", "~", "!", "^", "@"]
-    for idc, char in enumerate(spec):
-        if char in operators:
-            return spec[:idc]
-    return spec
-
-
-unpin_requirements = os.getenv("LOCK_REQUIREMENTS") == "0"
```

Comment:
It looks like we rely on `LOCK_REQUIREMENTS` on a lot of places and I am not sure how it would be with pyproject as it is a static file.

https://github.com/search?q=repo%3Aansible%2Fgalaxy_ng%20%20LOCK_REQUIREMENTS&type=code

Reply:
not sure, I need to research more

Reply:
actually, I realize both `DJANGO_ANSIBLE_BASE_BRANCH` and `LOCK_REQUIREMENTS` aren't `pyproject.toml` issues, we can handle those at dev environment level